src/main/java/cn/nukkit/config/ServerSettings.java

```diff
@@ -62,6 +62,8 @@ public static class BaseSettings extends OkaeriConfig {
         int autosave = 6000;
         @Comment("nukkit.server.settings.baseSettings.saveUnknownBlock")
         boolean saveUnknownBlock = true;
+        @Comment("nukkit.server.settings.baseSettings.enableCommandBlocks")
```

Comment:
you should create a new group of settings to control the gameplay and create the corresponding language translations